view/frontend/layout/omise_manage_cards.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0"?>
+
+<page
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../../../../../../lib/internal/Magento/Framework/View/Layout/etc/page_configuration.xsd">
```

Comment:
@jacstn is it safe to use relative path here? `../../../../../../../lib/internal/Magento/Framework/View/Layout/etc/page_configuration.xsd`

Would it be any issue when it comes to install the plugin via manual (placing folder at `app/code/Omise/Payment` and composer (inside `vendor` folder).

Reply:
I have copied that from module in `vendor`, so probably it would work, but playing it safe, I did change it to absolute.